README.md

```diff
@@ -1,13 +1,6 @@
-# Medplum Hello World
+# Foo Clinical
 
-This example app demonstrates the following:
-
-- Creating a new React app with Vite and TypeScript
-- Adding Medplum dependencies
-- Adding basic URL routing
-- Using the Medplum client to search for FHIR resources
-- Using Medplum GraphQL queries
-- Using Medplum React controls to display FHIR data
+This is a sample EHR-like application using Medplum. Medplum is a developer platform for medical apps.
 
```

Comment:
Generally, adding a table of contents is a good idea. It will structure your thinking more and make the future viewer aware of what is on the page